app/controllers/bookingsync_portal/admin/remote_accounts_controller.rb

```diff
@@ -9,6 +9,7 @@ def new
 
       def create
         @remote_account = scope.create(params_remote_account)
+        synchronize_bookingsync_source_id if current_account.synced_source_id.blank?
```

Comment:
I guess best to make it a service indeed or some decorator, like `BookingsyncPortal::Account`, might be better in terms of code organization.